processor/processor_test.go

```diff
@@ -2407,3 +2407,103 @@ var _ = Describe("TestSubJobMerger", func() {
 		})
 	})
 })
+
+var _ = Describe("TestConfigFilter", func() {
+	Context("testing config filter", func() {
+		It("success-full test", func() {
+			intgConfig := backendconfig.DestinationT{
+				ID:   "1",
+				Name: "test",
+				Config: map[string]interface{}{
+					"config_key":   "config_value",
+					"long_config1": "long_config1_value..................................",
+					"long_config2": map[string]interface{}{"hello": "world"},
+				},
+				DestinationDefinition: backendconfig.DestinationDefinitionT{
+					Config: map[string]interface{}{
+						"configFilters": []interface{}{"long_config1", "long_config2"},
+					},
```

Comment:
At least once we should use the outcome of `json.Unmarshal` in our tests, by unmarshalling a json string.

Reply:
Addressed